dotnet/src/SemanticKernel.Abstractions/Data/RecordAttributes/VectorStoreRecordDataAttribute.cs

```diff
@@ -21,6 +21,11 @@ public sealed class VectorStoreRecordDataAttribute : Attribute
     /// </summary>
     public bool IsFilterable { get; init; }
 
+    /// <summary>
+    /// Gets or sets a value indicating whether this data property is full text searchable.
```

Comment:
For folks looking at the API documentation should we include what the default value is in the API doc? Same would apply for the other properties.

Reply:
Makes sense.  Updated this and the other properties with more information.